docs/canvas/canvas-elements.asciidoc

```diff
@@ -1,91 +1,111 @@
 [role="xpack"]
-[[add-canvas-elements]]
-=== Add elements
+[[edit-workpads]]
+== Edit workpads
 
-Create a story about your data by adding elements to your workpad that include images, text, charts, and more. You can create your own elements and connect them to your data sources, add saved objects, and add your own images.
+To create the look and feel that you want, apply format settings to the entire workpad, or individual elements.
 
 [float]
-[[create-canvas-element]]
-==== Create an element
+[[create-variables]]
+=== Create variables
 
-Choose the type of element you want to use, then connect it to your own data.
+When you frequently use copy and paste, create variables to easily reuse strings and patterns. For example, when you clone a large workpad and need to connect your elements to a new index, use variables to update 
+each element instead of updating them manually.
 
-. Click *Add element*, then select the element you want to use.
-+
-[role="screenshot"]
-image::images/canvas-element-select.gif[Canvas elements]
+. Create the variables.
+.. Click *Add a variable*. 
+.. Specify the variable options, then click *Save changes*.
 
-. To familiarize yourself with the element, use the preconfigured data demo data.
-+
-By default, most of the elements you create use demo data until you change the data source. The demo data includes a small data set that you can use to experiment with your element.
+. Apply the variable syntax.
+.. Copy the variable syntax.
+.. Select the element you want to change, then open the expression editor. 
+.. Paste the syntax.
 
-. To connect the element to your data, select *Data*, then select one of the following data sources:
+For example, to change the index pattern for a set of charts:
 
-* *{es} SQL* &mdash; Access your data in {es} using SQL syntax. For information about SQL syntax, refer to {ref}/sql-spec.html[SQL language].
+Specify the variable syntax.
```

Comment:
Maybe this could be `Specify the variable options` ? I think of "variable syntax" being the expression syntax that gets copied to the clipboard